### PR TITLE
Feature/button component

### PR DESCRIPTION
**Pull request by markbolier**

### CODE REVIEW

File: src/app/planner/planner/planner.component.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
 <app-card *ngFor="let recipe of recipes; let i = index" [id]="recipe.id" [size]="recipe.size" [img]="recipe.img" [name]="recipe.name" (delete)="onDelete($event)">
```
Comment (Doenja) [inline src/app/planner/planner/planner.component.html:1]: In your ts you have addRecipe and onDelete. To keep things consistent you could choose a naming convention like addRecipe / deleteRecipe or onAdd / onDelete.

File: src/app/planner/card/card.component.html [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <div class="card-container">
 	<img src="{{img}}"/>
 	<div class="wrapper">
-		<h2 class="name">{{name}}</h2>
+		<h2 class="name"><ng-content></ng-content></h2>
 	</div>
 	<h2 class="size">{{size}}</h2>
 	<button class="delete-button" (click)="onClick(id)">x</button>
```
Comment (Doenja) [inline src/app/planner/card/card.component.html:7]: You could try and reuse your button component here. You might need an @Input() for the width. You can use ngClass to add dynamic classes.

**Review by Doenja — CHANGES_REQUESTED** [review]
Good job, very neat code 👍

File: src/app/planner/card/card.component.html [hunk 3]
```diff
@@ -1,8 +1,8 @@
 <div class="card-container">
-	<img src="{{img}}"/>
+	<img class="photo" src="{{img}}"/>
```
Comment (jarmovdh) [inline src/app/planner/card/card.component.html:2]: Other option to call your url: [src]="img"

**Review by jarmovdh — COMMENTED** [review]
Looks good mate🤟🏼

**Pull request merged**